ios/WeatherSensorModule.swift

```diff
@@ -0,0 +1,46 @@
+import Foundation
+import Combine
+
+@available(iOS 13.0, *)
+@objc(WeatherSensorModule)
+class WeatherSensorModule: RCTEventEmitter {
+  private var cancellable : Any?
+
+  public static var shared: WeatherSensorModule?
+    override init() {
+      super.init()
+      WeatherSensorModule.shared = self
+    }
+        
+    override func supportedEvents() -> [String]! {
+      return ["reading"]
+    }
+    
+  @available(iOS 13.0, *)
+  @objc
+  func collectReadings() -> Void {  
+    cancellable = WeatherSensorService.shared.reader.sensorReadingsPublisher
+      .sink{ value in 
```

Comment:
maybe I didn't notice this before, there should be a space between sink and { `sink {`